src/pages/Account/Account.js

```diff
@@ -0,0 +1,75 @@
+import { useState } from "react";
+import styles from "./Account.module.css";
+import { useAuth } from "../../context";
+import axios from "axios";
+
+export default function Account() {
+  const { user, updateAccountDetails } = useAuth();
+  const [isEditMode, setIsEditMode] = useState(false);
+  const [name, setName] = useState(user.name);
```

Comment:
Can use one state object with different properties, than using 3 different states. 
For eg - 
```jsx
const [userDetails, setuserDetails] = useState({
    username : "",
    email : "",
    password : ""
})
```

This will help lessen the complexity.